layouts/partials/openapi/resolve-ref-object.html

```diff
@@ -0,0 +1,74 @@
+{{/*
+
+Handles OpenAPI "Reference Objects".
+
+Reference objects are JSON objects with a single property, `$ref` (and
```

Comment:
It's actually a bit more complicated than that… Depending on the property it is used in, more fields might be overriden.

[This table](https://learn.openapis.org/referencing/overview.html#a-taxonomy-of-references) summarizes things.

Reply:
Right. I guess we are *specifically* targetting "reference objects" here, so it's correct that `summary` and `description` are preserved?

I guess we'd need to make this logic more generic if we wanted to use it in other places, but that could be a future change.

Reply:
Sure, what made me react here is more the comment in the resolve-refs partial that says it should be replaced by this one, but in its current state it's not possible.

Reply:
ah ok, maybe we can clarify that comment.